### ISSUE TITLE
Egg updates get synced without wisp restart

### ISSUE THREAD

**Issue by 0xCiBeR** [wisp-gg/frontend#429]
### Describe the feature and why you would like to see it.

Currently when you make changes to a egg, you need to go through every node and run wisp restart so it pulls the latest changes. This from an operational standpoint is a nightmare when the amount of nodes increases.

The daemon should be able to talk to the panel and get these changes without a wisp restart needed 

### Describe the solution you'd like.

Allow the daemon to poll for egg changes instead of requiring wisp restart ok the node.

**Comment by joti-prime** [comment]
Yes, will be interesting to have the option to restart each node and all together.

**Comment by smolf** [comment]
Yes please.
